tests/unit/test_mjolnir.py

```diff
@@ -21,15 +21,16 @@
 
 @patch.object(ops.JujuVersion, "from_environ")
 def test_update_peer_data_no_secrets(
-    mock_juju_env, harness_server_name_configured: Harness, monkeypatch: pytest.MonkeyPatch
+    mock_juju_env, harness: Harness, monkeypatch: pytest.MonkeyPatch
 ) -> None:
     """
     arrange: start the Synapse charm, set server_name, mock container and create_admin_user.
     act: call _update_peer_data.
     assert: relation data is updated with access token.
     """
+    harness.update_config({"enable_mjolnir": True})
+    harness.begin_with_initial_hooks()
```

Comment:
@amandahla I have also adapt the mjolnir tests to align with the reworked fixtures. Can you take a quick look? Tests seem to run fine on my end.